org-code-javabuilder/lib/src/main/java/dev/javabuilder/LocalMain.java

```diff
@@ -0,0 +1,30 @@
+package dev.javabuilder;
+
+import org.code.javabuilder.CodeBuilder;
+import org.code.javabuilder.InternalFacingException;
+import org.code.javabuilder.UserFacingException;
+
+/**
+ * This is a local version of the Javabuilder lambda function. The LocalInputAdapter can be used to
+ * pass input to the program. The "MyClass.java" program in the resources folder is the "user
+ * program." Output goes to the console.
+ */
+public class LocalMain {
+  public static void main(String[] args) {
+    final LocalInputAdapter inputAdapter = new LocalInputAdapter();
+    final LocalOutputAdapter outputAdapter = new LocalOutputAdapter(System.out);
+    final LocalProjectFileManager fileManager = new LocalProjectFileManager();
+    // Create and invoke the code execution environment
+    try {
+      CodeBuilder codeBuilder = new CodeBuilder(inputAdapter, outputAdapter, fileManager);
+      codeBuilder.compileUserCode();
+      codeBuilder.runUserCode();
+      codeBuilder.cleanUp();
```

Comment:
If it's important that cleanUp is always run, it should probably go in a finally block.  Alternatively, you can have CodeBuilder implement AutoCloseable and use java's try-with-resources idiom.

Reply:
Done. Thank you!